berkeley-function-call-leaderboard/bfcl/eval_checker/model_metadata.py

```diff
@@ -665,6 +695,11 @@
     "command-r-plus-FC-optimized": 3,
     "command-r-plus-optimized": 3,
     "yi-large-fc": 3,
+    "deepseek-ai/DeepSeek-V2-Chat-0628": 0.14,
```

Comment:
This should be removed. OSS models don't have a cost.

Reply:
Removed costs for both parts, sorry about that, I was looking at the API costs!